packages/next/README.md

```diff
@@ -568,8 +580,8 @@ Example:
    <Link href="/post?slug=something" as="/post/something">
    ```
 
-   - `href`: the path inside `pages` directory
-   - `as`: the path used by your server routes
+   - `href`: the path inside `pages` directory. Only right side of `?` character could change dynamically during execution, if needed, as left side is a filesystem path.
```

Comment:
Let's leave this one as-is since it's not related to dynamic routing.

Reply:
I agree. Changes removed in https://github.com/zeit/next.js/pull/8110/commits/d217c142acb3a587a555497ad7443ecc12c55367